docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+- **malicious broadcasts**: counts all blocks that are determined to be malicious
+
+
+#### Transaction propagation
+
+Transaction propagation tracks the sending and receiving of transactions on the peer-to-peer network. This includes:
+
+- **ingress announcements**: counts inbound announcements (notifications of a transaction's availability)
+- **known announcements**: counts announcements that are ignored because the local node is already aware of them
+- **underpriced announcements**: counts all announcements that never get fetched because they pay too little gas 
+- **malicious announcements**: counts announcements that are dropped because they appear malicious 
+- **ingress broadcasts**: counts all transactions fetched from peers
+- **known broadcasts**: counts transactions that are ignored because they duplicate transactions that the local node already knows about
+- **underpriced broadcasts**: counts all fetched transactions that are dropped due to paying insufficient gas
+- **otherreject broadcasts**: counts transactions that are rejected for reasons other than paying too little gas
+- **finished requests**: counts the successful delivery of a transaction, meaning it has been added to the local transaction pool
+- **failed requests**: counts the number of failed transaction deliveries, e.g. because a peer disconnected unexpectedly
+- **timed out requests**: counts the number of transaction requests that time out
+- **ingress replies**: tracks the total number of inbound replies to requests for transactions
+- **known replies**: tracks number of replies that are dropped because they are already known to the local node
+- **underpriced replies**: counts the number of replies that get dropped due to paying too little gas
+- **otherreject replies**: coutns the number of replies to transaction requests that get dropped for reasons other than paying too little gas
+
+![The tx propagation panel](/images/docs/grafana/tx-propagation.png)
+
+#### Block forwarding
+
+The block forwarding panel counts the announcements and the blocks that the local node receives that it should pass on to its peers. 
+
+#### Transaction fetcher peers
+
+The transaction fetcher peers panel shows how many peers the local node is connected to that can serve requests for transactions. The adjacent transaction fetcher hashes panel shows how many transaction hashes are available for fetching. Three statuses are reported in each panel: Waiting, queuing and fetching.
+
+#### Reorg
+
+The `reorg meter` panel simply counts the blocks added and the blocks removed during chain reorgs. The adjacent `Reorg total` panel shows the total number of reorg executions including both additions and removals.
+
+#### Goroutines
+
+Tracks the total number of active goroutines being used by Geth. Goroutines are lighweight threads managed by the Go runtime, they allow processes to 
```

Comment:
Ah, we should move it to System category?

Reply:
done!